test/conftest.py

```diff
@@ -338,6 +359,31 @@ def get_renate_module_mlp_and_data(
     return model, train_dataset, test_data
 
 
+@pytest.helpers.register
+def get_renate_module_mlp_data_and_loss(
```

Comment:
I would replace the previous helper with this to always return the loss function alongside.